backend/migrations/20200801122150-buildings-add.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  return db.createTable("buildings", {
```

Comment:
@gulyaasjaanos pls use renameTable instead
https://db-migrate.readthedocs.io/en/latest/API/SQL/#renametabletablename-newtablename-callback

Reply:
'buildings' is a new table

Reply:
oh, okay :D